changelog.d/20211221_182204_jimPruyne_caching_and_polling.rst

```diff
@@ -0,0 +1,19 @@
+.. A new scriv changelog fragment.
+..
+.. Uncomment the header that is right (remove the leading dots).
+..
+.. Features
+.. --------
+..
+.. - A bullet item for the Features category.
+..
+.. Bugfixes
+.. --------
+..
+.. - A bullet item for the Bugfixes category.
+..
+ Documentation
+ -------------
```

Comment:
It looks like there is a space character at the beginning of these two lines.

Reply:
good catch